qbit_manage.py

```diff
@@ -14,11 +15,13 @@
     print("Requirements Error: Requirements are not installed")
     sys.exit(0)
 
+REQUIRED_VERSION = (3, 8, 1)
+current_version = sys.version_info
```

Comment:
Since `REQUIRED_VERSION`  is a tuple you will need to convert it to a string.

```suggestion
current_version = sys.version_info
required_version_str = '.'.join(str(x) for x in REQUIRED_VERSION)

if current_version < (REQUIRED_VERSION):
	print(
		"Version Error: Version: %s.%s.%s incompatible with qbit_manage please use Python %s+"
		% (current_version[0], current_version[1], current_version[2], required_version_str)
	)
```